pom.xml

```diff
@@ -11,16 +11,16 @@
   <name>auth-code-notification</name>
   <properties>
     <java.version>11</java.version>
-    <spring-boot-dependencies.version>2.6.1</spring-boot-dependencies.version>
-    <spring-boot-maven-plugin.version>2.6.1</spring-boot-maven-plugin.version>
+    <spring-boot-dependencies.version>2.6.15</spring-boot-dependencies.version>
```

Comment:
Some Spring vulnerabilities are still being reported. Was the intention of this PR to address them all?

Reply:
Not all - the ones addressed are in spring-core and one in spring-web, but not all in spring- web.